sc-tools/sc-server/src/sc-server-impl/sc-memory-json/sc-memory-json-action/sc_memory_create_elements_by_scs_json_action.hpp

```diff
@@ -35,7 +35,16 @@ class ScMemoryCreateElementsByScsJsonAction : public ScMemoryJsonAction
       if (m_helper == nullptr)
         m_helper = new SCsHelper{*context, std::make_shared<DummyFileInterface>()};
 
-      sc_bool result = m_helper->GenerateBySCsText(scs);
+      sc_bool result = SC_FALSE;
+
+      try
+      {
+        result = m_helper->GenerateBySCsText(scs);
+      }
+      catch (utils::ScException const & e)
+      {
+      }
```

Comment:
Shouldn't we log the problem if we encountered any? Or is it logged somewhere else?